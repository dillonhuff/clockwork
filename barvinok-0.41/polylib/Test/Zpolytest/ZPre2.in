# Preimage by readMat["(i->3i+1,2i+5i)"] of readMat["(i,j->2i,2i+j)"]
M 2
3 3
2 0 0 
0 1 0
0 0 1
3 2
3 1
7 0
0 1
#readDom["{i,j| 1<= i,j<=10}"]
D 1
4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 100
          1 0 -1 100
F 11



