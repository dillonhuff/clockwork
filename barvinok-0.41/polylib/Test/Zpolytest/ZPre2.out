
ZPOLYHEDRON: Dimension 1 

LATTICE: 
2 2
    2    1
    0    1
POLYHEDRON Dimension:1
           Constraints:2  Equations:0  Rays:2  Lines:0
Constraints 2 3
Inequality: [    3    1 ]
Inequality: [  -14   93 ]
Rays 2 3
Vertex: [   -1 ]/3
Vertex: [   93 ]/14
